src/dsp_tools/dsp_tools.py

```diff
@@ -54,29 +51,44 @@ def make_parser() -> argparse.ArgumentParser:
     default_sipi = "http://0.0.0.0:1024"
 
     # make a parser
-    parser = argparse.ArgumentParser(description=f"DSP-TOOLS (version {version('dsp-tools')}, © {datetime.datetime.now().year} by DaSCH)")
-    subparsers = parser.add_subparsers(title="Subcommands", description="Valid subcommands are", help="sub-command help")
+    parser = argparse.ArgumentParser(
+        description=f"DSP-TOOLS (version {version('dsp-tools')}, © {datetime.datetime.now().year} by DaSCH)"
+    )
+    subparsers = parser.add_subparsers(
+        title="Subcommands", description="Valid subcommands are", help="sub-command help"
+    )
```

Comment:
IMO this got less readable

Reply:
This is the only linting choice of black that I don't like. But since I love the rest, I will have to live with this here...